io.openems.edge.battery.fenecon.home/src/io/openems/edge/battery/fenecon/home/ChannelIdImpl.java

```diff
@@ -0,0 +1,26 @@
+package io.openems.edge.battery.fenecon.home;
+
+import io.openems.edge.common.channel.ChannelId;
+import io.openems.edge.common.channel.Doc;
+
+public class ChannelIdImpl implements ChannelId {
```

Comment:
I prefer `public enum ChannelId implements io.openems.edge.common.channel.ChannelId {`

Why do you have this file?